util/dockerexporter/dockerexporter.go

```diff
@@ -16,20 +16,22 @@ import (
 	"github.com/pkg/errors"
 )
 
-// DockerExporter implements exporting to
+// DockerExporter implements containerd/images.Exporter to
 // Docker Combined Image JSON + Filesystem Changeset Format v1.1
 // https://github.com/moby/moby/blob/master/image/spec/v1.1.md#combined-image-json--filesystem-changeset-format
 // The outputed tarball is also compatible wih OCI Image Format Specification
 type DockerExporter struct {
-	name string
+	Name string
 }
 
+var _ images.Exporter = &DockerExporter{}
```

Comment:
what's this for?

Reply:
It's a cunning trick which checks (at compile time) that the `DockerExporter` type implements the `images.Exporter` interface.